From bbc825e93df70c934d4fc646af04440e651cb29c Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sat, 29 Jan 2022 01:02:42 +0200
Subject: [PATCH] [SW-65638] gaudi2: release: special ipmi dryrun

Allows skipping the ipmi commands but still logs it out

Change-Id: I67441b314379e0a5f52d55b2cca0a10e45f0213d
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/scripts/fwi/fw_installer.sh b/scripts/fwi/fw_installer.sh
index 89c807b..c70e4b5 100755
--- a/scripts/fwi/fw_installer.sh
+++ b/scripts/fwi/fw_installer.sh
@@ -1182,6 +1182,9 @@
 		-k|--ignore-ssh-init)
 			ignore_ssh_init_f=1
 			;;
+		-P|--ignore-ctrl-proto)
+			ignore_ctrl_proto_f=1
+			;;
 		-I)
 			ignore_args=(${args[@]})
 			;;
diff --git a/scripts/fwi/ipmitool.rc b/scripts/fwi/ipmitool.rc
index b3ed435..7561607 100644
--- a/scripts/fwi/ipmitool.rc
+++ b/scripts/fwi/ipmitool.rc
@@ -2,6 +2,11 @@
 {
 	local err
 	local ret
+
+	plog -l 6 "INFO: [${ignore_ctrl_proto_f:-0}] SENDING =>  ipmitool $*"
+
+	[ ${ignore_ctrl_proto_f:-0} -eq 1 ] && return 0
+
 	ret=($(ipmitool $*))
 	err=$?
 	[ $err -eq 0 ] && {
